lib/signs/utilities/headways.ex

```diff
@@ -36,8 +42,24 @@ defmodule Signs.Utilities.Headways do
   defp get_stop_ids(sign, nil), do: Signs.Utilities.SourceConfig.sign_stop_ids(sign.source_config)
   defp get_stop_ids(sign, config), do: [sign.headway_stop_id || config.stop_id]
 
-  @spec get_destination(SourceConfig.source() | nil) :: PaEss.destination() | nil
+  def get_headway_destination(config) do
+    get_destination(config)
+  end
+
+  @spec get_destination(SourceConfig.source() | list(SourceConfig.source()) | nil) ::
+          PaEss.destination() | nil
   defp get_destination(nil), do: nil
+
+  defp get_destination(config) when is_list(config) do
+    config
+    |> Enum.map(& &1.headway_destination)
+    |> Enum.uniq()
+    |> case do
+      [destination] -> destination
```

Comment:
Well, I added this logic mostly as a defensive measure since there's not any validation against differing headway destinations between individual sources. There's no cases currently where we'd want them to differ, but in a case where that might happen accidentally (as I did with some of the GLX stops recently 😬) it would be better imo to have the code default to just "trains", rather than assume that both sources have the same destination. That said, I think there _could_ be potential that this might become a legitimate use case, particularly at transfer hubs or busways where there might be a lot of destinations to consider, and once we have the ability to put both rail and bus predictions on a single sign. Although as I mentioned in another comment, I'm not sure if bus headways are a very well-defined and rider-helpful thing at the moment.

And yeah I think ideally, we'd want the headway destination to be defined a level up because headway mode is meant to act as a general default for the sign when there's an absence of predictions, although that is somewhat changing with this PR.